MusiciansAPP/MusiciansAPP.DAL/MusiciansAPP.DAL.csproj

```diff
@@ -6,6 +6,10 @@
 		<Nullable>disable</Nullable>
 	</PropertyGroup>
 
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
Comment to the 18th line.
DAL should not reference BL project.
This is the idea: higher layers depend on lower layers, but not vice versa.